.gitignore

```diff
@@ -2,6 +2,7 @@
 build
 coverage
 cypress
+hooks
```

Comment:
What is this for?

Reply:
We build `wp.hooks` file here: `hooks/build/index.js` as you discovered. I should mention that explicitly.